protos/kvstore.proto

```diff
@@ -4,6 +4,7 @@ package kvstore;
 message PutRequest {
   bytes key = 1;
   bytes value = 2;
+  bytes filename = 3;
```

Comment:
This may not be necessary as the filename should be passed into the kvstore at the command line.

Reply:
Yep. Totally misunderstood instructions. Thanks for pointing it out